pybullet_toolkit/robots/ur5_rg2.py

```diff
@@ -30,14 +30,20 @@ def reset(self):
     ur5_urdf_filepath = os.path.join(self.root_dir, 'urdf/ur5/ur5_w_simple_gripper.urdf')
     self.id = pb.loadURDF(ur5_urdf_filepath, [0,0,0], [0,0,0,1])
     self.is_holding = False
+    self.gripper_closed = False
     self.num_joints = pb.getNumJoints(self.id)
     [pb.resetJointState(self.id, idx, self.home_positions[idx]) for idx in range(self.num_joints)]
 
     self.motor_names = list()
     self.motor_indices = list()
+    self.arm_joint_names = list()
+    self.arm_joint_indices = list()
     for i in range (self.num_joints):
       joint_info = pb.getJointInfo(self.id, i)
       q_index = joint_info[3]
+      if i in range(1, 7):
+        self.arm_joint_names.append(str(joint_info[1]))
+        self.arm_joint_indices.append(i)
       if q_index > -1:
```

Comment:
I think we can get rid of the motor indices/names and just replace them with arm and gripper indices/names.